models/rbac.authorization.k8s.io.roletemplate.js

```diff
@@ -0,0 +1,228 @@
+import { uniq } from '@/utils/array';
+import Vue from 'vue';
+
+export const RESOURCES = [

```

Comment:
This list should come from schemas (`attributes.kind`), not be hardcoded into the UI.

Reply:
Done.